Resources/config/services.yml

```diff
@@ -16,23 +16,29 @@ services:
             - { name: form.type_extension, alias: form }
 
 
+    becklyn_rad.service.simple_route_voter:
+        class: Becklyn\RadBundle\Service\SimpleRouteVoter
+        arguments:
+            - "@request_stack"
```

Comment:
This change is not correct and probably even a BC.

Please take a look at lines 4-10 of this exact file. There was a reason to leave it commented. 😉